src/controllers/notifications.js

```diff
@@ -71,6 +73,16 @@ const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
 exports.post = function * postNotification () {
   const notification = yield requestUtil.validateBody(this, 'Notification')
   log.debug('Got notification: ' + JSON.stringify(notification))
+
+  const verifySignature = this.config.get('notifications.must_verify')
+  if (verifySignature) {
+    const result = model.verifySignature(notification, this.config)
+    if (!result.valid) {
+      log.warn('Signature verification failed: ' + result.error)
+      throw new UnprocessableEntityError('Notification failed signature verification')
```

Comment:
Would this cause the scalability problem we have addressed recently?  
1. Ledger -> Connector : notification with wrong signature
2. Ledger <- Connector : 422 because the signature is wrong
3. Repeat from step 1. 
Flooding the servers and impacting the performance negatively?